test/account.spec.ts

```diff
@@ -16,6 +18,187 @@ import {
+        'c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
+      )
+    })
+  })
+
+  describe('from RLP data', function() {
+    const accountRlp = Buffer.from(
+      'f84602820384a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a0c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
+      'hex',
+    )
+    const account = Account.fromRlpSerializedAccount(accountRlp)
+    it('should have correct nonce', function() {
+      assert.ok(account.nonce.eqn(2))
+    })
+    it('should have correct balance', function() {
+      assert.ok(account.balance.eqn(900))
+    })
+    it('should have correct stateRoot', function() {
+      assert.equal(
+        account.stateRoot.toString('hex'),
+        '56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
+      )
+    })
+    it('should have correct codeHash', function() {
+      assert.equal(
+        account.codeHash.toString('hex'),
+        'c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
+      )
+    })
+  })
+
+  describe('serialize', function() {
+    const raw = {
+      nonce: '0x01',
+      balance: '0x42',
+      stateRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
+      codeHash: '0xc5d2461236f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
+    }
+    const account = Account.fromAccountData(raw)
+    const accountRlp = rlp.encode([raw.nonce, raw.balance, raw.stateRoot, raw.codeHash])
+    it('should serialize correctly', function() {
+      assert.ok(account.serialize().equals(accountRlp))
+    })
+  })
+
+  describe('isContract', function() {
+    it('should return false for a non-contract account', function() {
+      const accountRlp = Buffer.from(
+        'f84602820384a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a0c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
+        'hex',
+      )
+      const account = Account.fromRlpSerializedAccount(accountRlp)
+      assert.equal(account.isContract(), false)
+    })
+
+    it('should return true for a contract account', function() {
+      const raw = {
+        nonce: '0x01',
+        balance: '0x0042',
+        stateRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
+        codeHash: '0xc5d2461236f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
+      }
+      const account = Account.fromAccountData(raw)
+      assert.ok(account.isContract())
+    })
+  })
+
+  describe('isEmpty', function() {
+    it('should return true for an empty account', function() {
+      const account = new Account()
+      assert.ok(account.isEmpty())
+    })
+
+    it('should return false for a non-empty account', function() {
+      const raw = {
+        nonce: '0x01',
+        balance: '0x0042',
+        stateRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
+        codeHash: '0xc5d2461236f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470',
```

Comment:
This test is confusing, the `codeHash` looks like `KECCAK256_NULL` yet it is not (by a few characters)

Reply:
updated in https://github.com/ethereumjs/ethereumjs-util/pull/275/commits/36a070008a4048461d315db487cfbe9f83e16b24